core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -90,6 +90,7 @@
           .put(PropertyKey.WORKER_TIERED_STORE_LEVEL1_DIRS_QUOTA, "2GB")
           .put(PropertyKey.WORKER_TIERED_STORE_LEVEL1_DIRS_PATH, mHddDir)
           .put(PropertyKey.WORKER_RPC_PORT, "0")
+          .put(PropertyKey.WORKER_MANAGEMENT_TIER_ALIGN_RESERVED_BYTES, "512MB")
```

Comment:
The default value of 1024M is not obvious enough , which will affect the allocation of blocks